wasi/src/main/java/com/dylibso/chicory/wasi/WasiPreview1.java

```diff
@@ -170,9 +170,26 @@ public HostFunction clockResGet() {
         return new HostFunction(
                 (Instance instance, Value... args) -> {
                     logger.info("clock_res_get: " + Arrays.toString(args));
-                    throw new WASMRuntimeException(
-                            "We don't yet support this WASI call: clock_res_get");
-                    // return new Value[] { Value.i32(0) };
+                    int clockId = args[0].asInt();
+                    int resultPtr = args[1].asInt();
+
+                    Memory memory = instance.memory();
+                    switch (clockId) {
+                        case WasiClockId.REALTIME:
+                            memory.writeLong(resultPtr, 1L);
```

Comment:
I have done some reading, but I cannot come up with anything better than "just use 1" ...
I'm open to suggestions!

Reply:
This seems fine to me.